### DISCUSSION TITLE
Is setting contentInsets externally for over-scroll behavior not supported?

### DISCUSSION THREAD

**Discussion by denil-ct** [krzyzanowskim/STTextView#13] [Questions]
Noticed this change [here](https://github.com/krzyzanowskim/STTextView/blob/2cd5d818b2b2057886402d9741288b2e1a58707d/Sources/STTextView/STTextView.swift#L667-L672) which will essentially make any content insets set externally to be zero as well. [Previously](https://github.com/krzyzanowskim/STTextView/blob/3df7ce1829a9277f427daa794e8a0d1aaeacfbb7/Sources/STTextView/STTextView.swift#L648-L653) it was made zero only for left inset. But after reverting the over-scroll changes its made zero for all sides. Is this intentional or an error? The issue is a [test](https://github.com/CodeEditApp/CodeEditTextView/blob/2cba05226f4be03472796ce9c5167fd76806bf08/Tests/CodeEditTextViewTests/STTextViewControllerTests.swift#L66-L87) written for over-scroll behavior on an older version (0.1.2) is now failing. Should it be disabled now as contentInsets property is not working as intended?

Also Readme still references contentInset property on textview.

**Comment by krzyzanowskim** [comment]
good catch. it looks like an invalid change revert. Fixed in 0.3.3

**Reply by denil-ct** [reply]
Thanks for the quick fix.

**Comment by thecoolwinter** [comment]
Is it intentional that setting left insets still aren't supported?

**Reply by krzyzanowskim** [reply]
please be more specific. there's no code that suppose to override the insets as far as I am concerned.

**Reply by thecoolwinter** [reply]
I was looking at an old release sorry, I saw that the now-removed `tile` function was setting the left inset of the clip view to `0`. Updated and works now, thanks!
